sdk/eventhub/Azure.Messaging.EventHubs/src/Amqp/AmqpError.cs

```diff
@@ -113,6 +113,26 @@ internal static class AmqpError
             return CreateException(error.Condition.Value, error.Description, eventHubsResource);
         }
 
+        /// <summary>
+        ///   Determines if a given AMQP message response is an error and, if so, throws the
+        ///   appropriate corresponding exception type.
+        /// </summary>
+        ///
+        /// <param name="response">The AMQP response message to consider.</param>
+        /// <param name="eventHubName">The name of the Event Hub associated with the request.</param>
+        ///
+        public static void ThrowIfErrorResponse(AmqpMessage response,
+                                                string eventHubName)
+        {
+            var statusCode = default(int);
+
+            if ((response?.ApplicationProperties?.Map.TryGetValue(AmqpResponse.StatusCode, out statusCode) != true)
```

Comment:
Just out of curiosity: any reason for preferring `!= true` over `== false`? One character less, maybe?

Reply:
No good reason; my brain went to the standard `if (!something)` and then had to expand because of the implicit conversion of the nullable.   It reads more clearly  as `== false`.   Changed.